src/index.css

```diff
@@ -0,0 +1,105 @@
+@import url('https://fonts.googleapis.com/css2?family=Barlow+Semi+Condensed:wght@500;600;700&family=Cairo:wght@200;300;400;500;600;700&family=Goudy+Bookletter+1911&display=swap');
+
+body {
+  margin: 0;
+  box-sizing: border-box;
+  padding: 0;
+  font-family: 'Barlow Semi Condensed', sans-serif;
+}
+
+.link.one {
+  margin-left: auto;
+}
+
+.link {
+  font-size: 20px;
+  text-decoration: none;
+  color: #222831;
+  padding: 0.4rem 0.4rem;
+  border-radius: 3px;
+}
+
+.link:hover {
+  text-decoration: underline;
+}
+
+.link.active {
+  color: #ffd369;
+  text-decoration: underline;
+  background-color: #393e4665;
+  filter: drop-shadow(0.25rem 0.25rem 0.5rem #222831);
+}
+
+.home {
+  scroll-behavior: smooth;
+}
+
+.racks {
+  background-color: #222831;
+}
+
+.header {
+  color: #ffd369;
+  font-family: 'Barlow Semi Condensed', sans-serif;
+  margin: 0 auto;
+  padding-top: 3rem;
+  font-size: 72px;
+  width: 75rem;
+}
+
+.movies-rack,
+.series-rack {
+  padding: 5rem 0 0;
+}
+
+.series-rack {
+  padding-bottom: 5rem;
+}
+
+.rack-head {
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
+  line-height: 5rem;
+  width: 75rem;
+  margin: 0 auto 1rem;
+}
+
+.rack-head h1 {
+  font-size: 60px;
+  color: #ffd369;
+  margin: 1rem 1rem 0 0;
+}
+
+.rack-head h2 {
+  font-family: 'Barlow Semi Condensed', sans-serif;
+  line-height: 0;
+  padding-top: 1rem;
+  font-size: 42px;
+  font-weight: 600;
+  color: #eee;
+}
+
+.arrow {
+  border-top: 5px solid #eee;
+  border-right: 5px solid #eee;
+  height: 1.3rem;
+  width: 1.3rem;
+  transform: rotate(45deg);
+  align-self: flex-end;
+  margin: 0 0 1.3rem 0.5rem;
+}
+
+.page-link {
+  text-decoration: none;
+  color: #eee;
+  font-size: 18px;
+  margin-left: 29rem;
+  align-self: flex-end;
+  line-height: 25px;
+}
+
+.page-link:hover {
+  text-decoration: underline;
+  color: #ffd369;
+}
```

Comment:
- [ ] As required by the previous reviewer, you should follow the provided design cause is a part of the project requirement, you can just personalize the content but the design is a part of the project requirements. Please kindly follow the original design 🙃 🙃. The importance of following the original design is that you can be sure that if the customers give you their design, you can provide a good and exact product wanted by the customer, and the customer will be satisfied with your work. A customer cannot give you a design they want and then you present them with another design. To be professional it's better to provide the same product asked 😁 😁  
- [ ] Also after redesigning your project kindly record a new video that presents a new project design 😁 😁